pypeit/core/gui/identify.py

```diff
@@ -2,6 +2,7 @@
 import copy
 import numpy as np
 import matplotlib
+matplotlib.use('Qt5Agg')
```

Comment:
If anything other than Qt5Agg is used then the identify gui fails to save the solution (see issue #1158), so I added this here.  I can take it out, if you prefer.  Maybe @rcooke-ast should comment?

Reply:
Thanks David - if you hadn't put it in, I would have likely submitted a one-line PR in the future with this line to fix the bug... I think it can stay @jhennawi.

Reply:
@finagle29 -- Please comment on whether this will work with both of our Qt packages.  Thx.

Reply:
it would be better to leave backend selection up to the user, but the QT dependence is so deeply baked in currently that i think this might as well be there. `Qt5Agg` should be abstract enough to work with either `pyqt5` or `pyside2`, but i haven't tested that personally to confirm.